wavenet_ops.py

```diff
@@ -35,3 +35,26 @@ def causal_conv(value, filter_, dilation, name='causal_conv'):
                          [0, 0, 0],
                          [-1, tf.shape(value)[1], -1])
         return result
+
```

Comment:
Only one line break is allowed before function.

(Sorry for pointing out only small things... 🙇)